docs/mochitests.md

```diff
@@ -139,9 +139,17 @@ add_task(function* () {
 
 The Debugger Mochitest API Documentation can be found [here](https://devtools-html.github.io/debugger.html/reference#mochitest).
 
+## Debugging Tips
+
+* Run your mochitest with `yarn mochid <filename>` and add a `debugger` statement to your test before or after a failure.
```

Comment:
screenshots of the test paused in the debugger code and the test code would help show what it is happening